internal/proto/controller/api/resources/hostcatalogs/v1/host_catalog.proto

```diff
@@ -14,50 +14,52 @@ import "controller/custom_options/v1/options.proto";
 // HostCatalog manages Hosts and Host Sets
 message HostCatalog {
 	// Output only. The ID of the host.
-	string id = 10;
+	string id = 10;  // @gotags: `class:"public"`
 
 	// The ID of the Scope of which this Host Catalog is a part.
-	string scope_id = 20 [json_name="scope_id"];
+	string scope_id = 20 [json_name="scope_id"];  // @gotags: `class:"public"`
 
 	// Output only. Scope information for this resource.
 	resources.scopes.v1.ScopeInfo scope = 30;
 
 	// The ID of the plugin of which this catalog is created.
-	string plugin_id = 34 [json_name="plugin_id", (custom_options.v1.generate_sdk_option) = true];
+	string plugin_id = 34 [json_name="plugin_id", (custom_options.v1.generate_sdk_option) = true];  // @gotags: `class:"public"`
 
 	// Output only. Plugin information for this resource.
-	resources.plugins.v1.PluginInfo plugin = 35;
+	resources.plugins.v1.PluginInfo plugin = 35;  // @gotags: `class:"public"`
 
 	// Optional name for identification purposes.
-	google.protobuf.StringValue name = 40 [(custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = {this:"name" that: "name"}];
+	google.protobuf.StringValue name = 40 [(custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = {this:"name" that: "name"}];  // @gotags: `class:"public"`
```

Comment:
Might be worth asking @jimlambrt how this will work. Generally only byte slices and strings will be classified and messages won't (but the contents may be) and this is a strange hybrid where if it's like a message this tag is unnecessary and if it's more like a string we should ensure that the classification library handles these proto types.